docs/articles/documentation/test-api/intercepting-http-requests/mocking-http-requests.md

```diff
@@ -97,7 +97,7 @@ var mock = RequestMock()
 var mock = RequestMock()
     .onRequestTo(/*...*/)
     .respond((req, res) => {
-        // ...
+        // a sync or async function
```

Comment:
`// a sync or async function` - it's not correct.
The handler `(req, res)` can be sync or async, not it's content.

In my opinion we just need to extend the example:
```js
The response function can be synchronous or asynchronous

var mock = RequestMock()
   .onRequestTo(/*...*/)
   .respond((req, res) => {
        res.setBody('<html><body><h1>This is a page</h1></body></html>');
    })
    .onRequestTo(/*...*/)
    .respond(async (req, res) => {
        const body = await fetch('https://web-site.com/route/data');
        res.setBody(body);
    });
```

Reply:
Sure it's handler. So you are saying someone may think that we mean 'asynchronous content' - okay.